### ISSUE TITLE
Implementing other Unity Wii Sports remakes?

### ISSUE THREAD

**Issue by CCIGAMES** [CCIGAMES/Rii-sports#72]
https://github.com/TomSchoolar/Wii-Bowling-Unity-
https://github.com/TheDevAtlas/Wii-Are-Resorting-To-Violence
https://github.com/kyleshepard/wii_bowling_2

these 3 are interesting

**Comment by CCIGAMES** [comment]
https://archive.org/details/wuhu-island-explorer/u8GLcM.png found this too, quite interesting

**Comment by CCIGAMES** [comment]
what r you gonna do @fgsoftware1 ?

**Comment by fgsoftware1** [comment]
> what r you gonna do @fgsoftware1 ?

I'm not having much time, but, I was trying make the whuhu island, but the model we got is kinda simple though and I don't know what to do there

**Comment by fgsoftware1** [comment]
@CCIGAMES you tried my bowling thing?

**Comment by CCIGAMES** [comment]
Yes, and it's ok, however I've just found someone's remake of Wii Bowling which is fully developed, also we are moving to the use of the Wii Sports decomp for the project, so the user has to supply a rom and Nintendo can't take us down, also my discord account is permanently disabled so github is my only way to talk to you

**Comment by fgsoftware1** [comment]
> Yes, and it's good, however I've just found someone's remake of Wii Bowling which is fully developed, also we are moving to the use of the Wii Sports decomp for the project, so the user has to supply a rom and Nintendo can't take us down, also my discord account is permanently disabled so github is my only way to talk to you

so you're planning making an emulator? in which language? ROMs ARE ILEGAL!

**Comment by CCIGAMES** [comment]
> > Yes, and it's good, however I've just found someone's remake of Wii Bowling which is fully developed, also we are moving to the use of the Wii Sports decomp for the project, so the user has to supply a rom and Nintendo can't take us down, also my discord account is permanently disabled so github is my only way to talk to you
> 
> so you're planning making an emulator? in which language? ROMs ARE ILEGAL!

not an emulator, it uses the C and Assembly decomp of Wii Sports as well as several C# unity remakes of the game and C++ from the Dolphin emulator and parts of other Wii Sports games like Wii Sports Resort etc. Roms arent illegal, as long as the user owns the game.

**Comment by fgsoftware1** [comment]
> > > Yes, and it's good, however I've just found someone's remake of Wii Bowling which is fully developed, also we are moving to the use of the Wii Sports decomp for the project, so the user has to supply a rom and Nintendo can't take us down, also my discord account is permanently disabled so github is my only way to talk to you
> > 
> > so you're planning making an emulator? in which language? ROMs ARE ILEGAL!
> 
> not an emulator, it uses the C and Assembly decomp of Wii Sports as well as several C# unity remakes of the game and C++ from the Dolphin emulator and parts of other Wii Sports games like Wii Sports Resort etc. Roms arent illegal, as long as the user owns the game.

Game ROMs are literally reverse engineering which is prohibited by law, and how you're to mix up c++ with c#? It doesn't work like that, but you can use an IL like C++/CLI even though you'll need to rewrite all the c++ code you need to expose to c#

**Comment by CCIGAMES** [comment]
> > > > Yes, and it's good, however I've just found someone's remake of Wii Bowling which is fully developed, also we are moving to the use of the Wii Sports decomp for the project, so the user has to supply a rom and Nintendo can't take us down, also my discord account is permanently disabled so github is my only way to talk to you
> > > 
> > > 
> > > so you're planning making an emulator? in which language? ROMs ARE ILEGAL!
> > 
> > 
> > not an emulator, it uses the C and Assembly decomp of Wii Sports as well as several C# unity remakes of the game and C++ from the Dolphin emulator and parts of other Wii Sports games like Wii Sports Resort etc. Roms arent illegal, as long as the user owns the game.
> 
> Game ROMs are literally reverse engineering which is prohibited by law, and how you're to mix up c++ with c#? It doesn't work like that, but you can use an IL like C++/CLI even though you'll need to rewrite all the c++ code you need to expose to c#

Reverse Engineering and emulation is legal, dumping your own rom is fine, pirating roms isnt.
C++ is only for a minority of the stuff, so dont worry.

@OneProblemAtATime and I have talked through this before

**Comment by CCIGAMES** [comment]
@fgsoftware1 what are you doing with project next?

**Comment by CCIGAMES** [comment]
> > @fgsoftware1 what are you doing with project next?
> 
> i'm kinda trying make multiplayer but i'm not having much time

are you making code from scratch?

**Comment by fgsoftware1** [comment]
> > > @fgsoftware1 what are you doing with project next?
> > 
> > 
> > i'm kinda trying make multiplayer but i'm not having much time
> 
> are you making code from scratch?

i'm gonna use netcode for gameobjects(the new mulriplayer unity API)

**Comment by CCIGAMES** [comment]
> > > > @fgsoftware1 what are you doing with project next?
> > > 
> > > 
> > > i'm kinda trying make multiplayer but i'm not having much time
> > 
> > 
> > are you making code from scratch?
> 
> i'm gonna use netcode for gameobjects(the new mulriplayer unity API)

Make sure you start implementing the Wii Sports Decomp into the project
